soroban-built-in-token-spec/tests/test.rs

```diff
@@ -0,0 +1,96 @@
+#![cfg(feature = "testutils")]
+
+use soroban_auth::{
+    testutils::ed25519::{generate, sign},
+    Signature,
+};
+use soroban_sdk::{contractimpl, contracttype, symbol, BigInt, BytesN, Env, IntoVal};
+
+mod token_contract;
+use token_contract::{Identifier, TokenClient, TokenMetadata};
+
+#[contracttype]
+pub enum DataKey {
+    Token,
+}
+
+fn get_contract_id(e: &Env) -> Identifier {
+    Identifier::Contract(e.get_current_contract().into())
+}
+
+fn get_token(e: &Env) -> BytesN<32> {
+    e.contract_data().get_unchecked(DataKey::Token).unwrap()
+}
```

Comment:
What's the motivation for keeping the logic for these functions here, rather than putting the logic inside TestContract?

Reply:
This is how we've been adding helper functions for contract. We could instead move the private functions into the contract and call into them using `Self::get_contract_id(&e)`  and `Self::get_token_addr(&e)` (can't use `get_token` because it'll conflict in the contract). What do you think?